openquake/calculators/postproc/aelo_plots.py

```diff
@@ -427,3 +429,23 @@ def plot_disagg_by_src(dstore, site_idx=0, update_dstore=False):
         dstore['png/disagg_by_src-All-IMTs.png'] = Image.open(bio)
     fig.tight_layout()
     return plt
+
+
```

Comment:
I would plot all sites, i.e. do not pass site_idx. In AELO there is a single site so it is okay, and other situations one always wants to plot all sites.

Reply:
I refactored the function to use all available sites (in this case just one, but this way the function can be easily reused for cases with multiple sites).